app/routes/datasets.js

```diff
@@ -16,21 +16,36 @@ export default class extends Route {
   model(params) {
     let dataset = this.modelFor('application').findBy('id', params.dataset_id);
     let yearcolumn = dataset.get('yearcolumn');
+    const prqlSchema = dataset.get('schemaname') === 'tabular' ? 'tabular' : 'mapc';
```

Comment:
Is there a reason why the `schemaname` attribute can't be the `prqlSchema` value outright?

Reply:
At one point some the `schemaname` values for geospatial datasets were null/empty so this was a workaround for this, but they all seem to have been updated now.